simplic-session/Simplic.Session/Model/Session.cs

```diff
@@ -52,19 +53,21 @@ public class Session
         /// <summary>
         /// Gets or sets all available tenant organizations
         /// </summary>
-        private IList<Tenant.Organization> Organizations
+        public IReadOnlyList<Tenant.Organization> Organizations
```

Comment:
I dont understand properly. you are creating a readonly list, but inside get you cast it to list every single time this property is used. do you realize how much of a performance penalty is this ?